src/components/sde/tests/Minimal/Minimal_Test++.cpp

```diff
@@ -6,28 +6,40 @@
 #include "sde_lib.h"
 #include "sde_lib.hpp"
 
-long long local_var;
 
-void mintest_init(void){
-    papi_sde::PapiSde sde("Min Example Code in C++");
+////////////////////////////////////////////////////////////////////////////////
+//------- Library example that exports SDEs
+
+class MinTest{
+    private:
+        long long local_var;
+
+	public:
+        MinTest();
+        void dowork();
+};
 
-    local_var=0;
+MinTest::MinTest(){
+    papi_sde::PapiSde sde("Min Example Code in C++");
+    local_var = 0;
```

Comment:
Very small suggestion, but I would change the order here of the code from:
```
papi_sde::PapiSde sde("Min Example Code in C++");
local_var = 0;
sde.register_counter("Example Event", PAPI_SDE_RO|PAPI_SDE_DELTA, local_var);
```
to 
```
local_var = 0;
papi_sde::PapiSde sde("Min Example Code in C++");
sde.register_counter("Example Event", PAPI_SDE_RO|PAPI_SDE_DELTA, local_var);
```
I believe it reads better. However, this is mainly just personal preference.